dla/dla.md

```diff
@@ -0,0 +1,12 @@
+# FirstName LastName (Unique ID): dla
+## Profile Pic: 
+## Call me by this name: daekyung
+## Cohort: July 2023
+## Email ID: lakoda@kookmin.ac.kr
+## LinkedIn: 
```

Comment:
It is a very good idea to have a professional profile on LinkedIn, if you explicitly do not want to make a linkedIn profile then reply to this comment by saying, "I do not want to create my LinkedIn profile"